src/quicr_server.cpp

```diff
@@ -43,9 +47,8 @@ QuicRServer::QuicRServer(std::shared_ptr<qtransport::ITransport> transport_in,
 std::shared_ptr<qtransport::ITransport>
 QuicRServer::setupTransport(RelayInfo& relayInfo)
 {
-  qtransport::LogHandler logger;
   return qtransport::ITransport::make_server_transport(
-    t_relay, transport_delegate, logger);
+    t_relay, transport_delegate, logHandler);
 }
```

Comment:
nit s/logHandler/log_handler ..

Reply:
Are you suggesting a name change to the clsss variable?